tests/integration.rs

```diff
@@ -311,6 +357,94 @@ mod tests {
         }
     }
 
+    impl DiskConfig for WindowsDiskConfig {
+        fn prepare_cloudinit(&self, _tmp_dir: &TempDir, _network: &GuestNetworkConfig) -> String {
+            String::new()
+        }
+
+        fn prepare_files(&mut self, tmp_dir: &TempDir, _network: &GuestNetworkConfig) {
+            let snapshot_cow_path =
+                String::from(tmp_dir.path().join("snapshot_cow").to_str().unwrap());
+
+            // truncate -s 1G /tmp/snapshot_cow
+            std::process::Command::new("truncate")
+                .args(&["-s", "1G"])
+                .arg(snapshot_cow_path.as_str())
+                .output()
+                .expect("Expect creating snapshot CoW image to succeed");
```

Comment:
You can do this in Rust directly. https://doc.rust-lang.org/std/fs/struct.File.html#method.set_len

Reply:
Ok I'll update this.